client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -32,6 +34,7 @@ interface Props {
 }
 
 export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) => {
+	let importSiteHostName = '';
```

Comment:
Let's think about having a fallback, in case the site name is not defined... like 'your site'